src/operator/convolution_v1-inl.h

```diff
@@ -344,9 +348,7 @@ class ConvolutionV1Op : public Operator {
                                              shape_dstunit_[1],
                                              shape_dstunit_[2] * nstep_);
     index_t required_size = scol.Size() + sdst.Size();
-    CHECK_GE(param_.workspace, required_size)
-      << "\nMinimum workspace size: " << required_size * sizeof(DType) << " Bytes\n"
-      << "Given: " << param_.workspace * sizeof(DType) << " Bytes";
+    param_.workspace = required_size;
```

Comment:
do you actually need to optimize this?

Reply:
No need because it's deprecated.

Reply:
I change it because I'm doing a global search for `param_.workspace` and changing all the occurrence.

Reply:
I mean why do you need to modify param_. workspace?

Reply:
Yes, there is no need to modify it. I've revised.